src/rosdiscover/models/image_transport.py

```diff
@@ -0,0 +1,23 @@
+from ..interpreter import model
+from loguru import logger
+
+
+@model('image_transport', 'republish')
+def republish(c):
+    args = c.args.split(' ')
+    intopic: str = 'in'
+    outopic: str = 'out'
+    img_format: str = 'raw'
+
+    for a in args:
+        if a in ["raw", "compressed", "theora"]:
```

Comment:
Super minor comment: You can change the list, which is mutable, to a tuple, which is immutable. In _theory_, this should be faster.

Reply:
Done.